testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveTransactionalTable.java

```diff
@@ -238,6 +238,63 @@ public void testReadInsertOnly(boolean isPartitioned, BucketingType bucketingTyp
         }
     }
 
+    @Test(groups = HIVE_TRANSACTIONAL, dataProvider = "partitioningAndBucketingTypeDataProvider", timeOut = TEST_TIMEOUT)
+    @Flaky(issue = "https://github.com/trinodb/trino/issues/4927", match = "Hive table .* is corrupt. Found sub-directory in bucket directory for partition")
+    public void testReadNonOrcInsertOnly(boolean isPartitioned, BucketingType bucketingType)
```

Comment:
`bucketingType.getHiveClustering(` seems to be missing, so bucketingType is effectively ignored.